kolibri/core/assets/src/views/sync/FacilityNameAndSyncStatus/index.vue

```diff
@@ -44,6 +44,11 @@
             {{ $tr('lastSync', { relativeTime: formattedTime(facility.last_successful_sync) }) }}
           </span>
         </template>
+        <KButton
+          :text="$tr('createSync')"
+          appearance="basic-link"
+          @click="manageSync"
```

Comment:
To make this portable across the facility and device plugins, we will probably want to just emit an event from the component here `$emit('manageSync')` here, and handle that event wherever this component is used. That way we can navigate different in the facility plugin vs the device plugin as needed.